pvp/common/loggers.py

```diff
@@ -420,7 +451,10 @@ def load_file(self, filename = None):
             table = file.root.derived_quantities.readout
             derived_data = table.read()
 
-        data_dict = {"waveform_data": waveform_data, "control_data": control_data, "derived_data": derived_data}
+            table = file.root.program_information.readout
+            program_data = table.read()
+
+        data_dict = {"waveform_data": waveform_data, "control_data": control_data, "derived_data": derived_data, "program_information": program_data}
```

Comment:
does this save the version in every row of the continuous data?

Reply:
no, it's saved only once at the beginning